cmd/serverless-init/log/log_test.go

```diff
@@ -37,3 +37,14 @@ func TestIsEnabledFalse(t *testing.T) {
 	assert.False(t, isEnabled("1"))
 	assert.False(t, isEnabled("FALSE"))
 }
+
+func TestIsInstanceTailingEnabled(t *testing.T) {
```

Comment:
Is it worth adding a unit test where `DD_AAS_INSTANCE_LOGGING_ENABLED` is not set at all?

Reply:
It already is with `""`. An unset env var in Go returns `""`

Reply:
Yes, an unset environment variable and an environment variable set to an empty string both return empty strings. But they are test different cases.